R/by.R

```diff
@@ -5,14 +5,14 @@
 #' @param .by `r lifecycle::badge("experimental")`
 #'
 #'   <[`tidy-select`][dplyr_tidy_select]> Optionally, a selection of columns to
-#'   temporarily group by using an inline alternative to [group_by()]. For
+#'   group by for just this operation, as an alternative to [group_by()]. For
```

Comment:
```suggestion
#'   group by for just this operation, functioning as an alternative to [group_by()]. For
```

For some reason it sounds kind of strange without this. Might need to reflow to line wrap it